RS-Project14/EmployeeUsingList/EmployeeUsingArray.java

```diff
@@ -0,0 +1,48 @@
+package employee;
+
+import java.util.*;
+
+public class EmployeeUsingArray {

```

Comment:
you can create an interface to declare the methods and that can implemented by both the classes EmployeeUsingArray/Map.